src/relay/backend/build_module.cc

```diff
@@ -491,8 +491,8 @@ class RelayBuildModule : public runtime::ModuleNode {
     auto lowered_funcs = executor_codegen_->GetIRModule();
 
     // No need to build for external functions.
-    if (lowered_funcs.find("ext_dev") != lowered_funcs.end()) {
-      lowered_funcs.Set("ext_dev", IRModule());
+    if (lowered_funcs.find(Target("ext_dev")) != lowered_funcs.end()) {
+      lowered_funcs.Set(Target("ext_dev"), IRModule());
```

Comment:
My suggestion got squashed because you're too fast :smile_cat: I was suggesting moving the duplicated instantiation out:
```suggestion
    Target ext_dev("ext_dev");
    if (lowered_funcs.find(ext_dev) != lowered_funcs.end()) {
      lowered_funcs.Set(ext_dev, IRModule());
```

Reply:
Oh, my bad, I just didn't commit the file 😅

Reply:
Oh, my bad, I just didn't commit the file 😅

Reply:
Ahh, classic :sweat_smile: